ConsoleApp/Program.cs

```diff
@@ -0,0 +1,135 @@
+﻿internal class Program 
+{
+    static int GetMaxValue(int a, int b)
+    {
+        Console.WriteLine ($"Method was called and it got follow parameters: a={a} b={b}");
+        return Math.Max(a, b);
+    }
+
+        static int GetMinValue(int a, int b)
+    {
+        Console.WriteLine ($"Method was called and it got follow parameters: a={a} b={b}");
+        return Math.Min(a, b);
+    }
+
+    static int GetMaxValue(int a, int b, int c)
+    {
+        Console.WriteLine ($"Method was called and it got follow parameters: a={a} b={b} c={c}");
+        return new int [] {a, b, c}.Max();
+    }
+
+    static int GetMinValue(int a, int b, int c)
+    {
+        Console.WriteLine ($"Method was called and it got follow parameters: a={a} b={b} c={c}");
+        return new int [] {a, b, c}.Min();
+    }
+
+    static int GetMaxValue(int a, int b, int c, int d)
+    {
+        Console.WriteLine ($"Method was called and it got follow parameters: a={a} b={b} c={c} d={d}");
+        return new int [] {a, b, c, d}.Max();
+    }
+
+   static int GetMinValue(int a, int b, int c, int d)
+    {
+        Console.WriteLine ($"Method was called and it got follow parameters: a={a} b={b} c={c} d={d}");
+        return new int [] {a, b, c, d}.Min();
+    }
+    // TrySumIfOdd
+
+    static bool TrySumIfOdd(int x, int y, out int sum)
+    {
+        sum = Math.Min(x , y);
+
+        while (x != y)
+        {
+            sum += (x > y) ? ++y : ++x;
+        }
+
+        if (sum % 2 != 0) 
```

Comment:
you can simplify here to just `return sum % 2 != 0`

Reply:
wow)) cool))